src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -383,8 +383,10 @@ See the accompanying LICENSE file for applicable license.
   
   <!-- section processor - div with no generated title -->
   <xsl:template match="*[contains(@class, ' topic/section ')]" name="topic.section">
-    <section class="section">
-      <xsl:call-template name="commonattributes"/>
+    <section>
+      <xsl:call-template name="commonattributes">
+        <xsl:with-param name="default-output-class" select="'section'"/>
```

Comment:
Several templates do this (I noted on a couple others) - where we previously set class on the element directly, then called commonattributes, we now just call common attributes but with the default. I think that the previous way of doing it almost made the original setting a no-op, because `commonattributes` resets class entirely, unless you override it to skip setting class. In that case - where you override `commonattributes` to avoid setting a class on `section` - the net result here is just to remove `@class` right? 

With a 4.0 release I'm not really worried about compatibility per-se, just want to make sure we do what is the best design. Might be better to delete the class as you've done, but without passing in `section` as a default, as that should already be set (by default) due to class ancestry